pkg/crd/kubermatic/v1/cluster.go

```diff
@@ -175,6 +175,9 @@ type ClusterSpec struct {
 
 	// CNIPlugin contains the spec of the CNI plugin to be installed in the cluster.
 	CNIPlugin *CNIPluginSettings `json:"cniPlugin,omitempty"`
+
+	// IsKonnectivityEnabled enables connectivity for controlplane to node network communication.
+	IsKonnectivityEnabled bool `json:"isKonnectivityEnabled,omitempty"`
```

Comment:
I would prefer to put this into `ClusterNetwork` (`ClusterNetworkingConfig`).
Also, I would prefer to keep the naming convention of the bool flags in the API - without the `Is` prefix (e.g. `EnableUserSSHKeyAgent` / `MonitoringEnabled` / `NodeLocalDNSCacheEnabled`).